build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/dsl/CodeStyle.kt

```diff
@@ -19,34 +18,33 @@ import org.gradle.kotlin.dsl.register
 fun Project.configureLinting(lintPaths: List<String>) {
     verifyRootProject { "Kotlin SDK lint configuration is expected to be configured on the root project" }
 
-    val ktlint = configurations.create("ktlint") {
-        attributes {
-            attribute(Bundling.BUNDLING_ATTRIBUTE, objects.named(Bundling.EXTERNAL))
-        }
-    }
+    val ktlint by configurations.creating
 
-    // TODO - is there anyway to align this with the version from libs.versions.toml in this project/repo
-    val ktlintVersion = "0.48.1"
     dependencies {
-        ktlint("com.pinterest:ktlint:$ktlintVersion")
+        val ktlintVersion = "1.3.0"
+        ktlint("com.pinterest.ktlint:ktlint-cli:$ktlintVersion") {
+            attributes {
+                attribute(Bundling.BUNDLING_ATTRIBUTE, objects.named(Bundling.EXTERNAL))
+            }
+        }
     }
 
     // add the buildscript classpath which should pickup our custom ktlint-rules (via runtimeOnly dep on this plugin)
     // plus any custom rules added by consumer
-    val execKtlintClaspath = ktlint + buildscript.configurations.getByName("classpath")
+    val execKtlintClasspath = ktlint + buildscript.configurations.getByName("classpath")
+
     tasks.register<JavaExec>("ktlint") {
+        group = LifecycleBasePlugin.VERIFICATION_GROUP
         description = "Check Kotlin code style."
-        group = "Verification"
-        classpath = execKtlintClaspath
+        classpath = execKtlintClasspath
         mainClass.set("com.pinterest.ktlint.Main")
         args = lintPaths
-        jvmArgs("--add-opens", "java.base/java.lang=ALL-UNNAMED")
```

Comment:
Just following the guide on ktlint docs: https://pinterest.github.io/ktlint/latest/install/integrations/#custom-gradle-integration-with-kotlin-dsl

Reply:
I would lean towards keeping it since we don't know if it was necessary but I don't feel strongly about it. A good way to test this would be to run ktlint twice, once with and once without

Reply:
Sorry, just saw this comment after I already merged the PR. I had already tested with and without the jvmArgs locally, there was no difference. In general I think we should trust our dependencies' onboarding guides and if they end up being incorrect, send them a PR to fix the documentation.